src/stores/game.js

```diff
@@ -4,6 +4,15 @@ import { cloneDeep } from 'lodash';
 import { io } from '@/plugins/sails.js';
 import { sleep } from '../util/sleep';
 
+/**
+ * Throws an error if VITE_USE_GAMESTATE_API is enabled
+ */
+const checkGameStateApi = () => {
+  if (import.meta.env.VITE_USE_GAMESTATE_API === 'true') {
+    throw 'This action is not supported yet in GameState API';
+  }
+};
+
```

Comment:
I don't believe we need to check for an error here anymore after updating the way the API is consumed in `makeSocketRequest`.